### DISCUSSION TITLE
重庆联通阻断Hysteria2协议的流量

### DISCUSSION THREAD

**Discussion by PekingJam** [apernet/hysteria#1249] [General]
服务端：debian 12，内核：6.1.0-26-cloud-amd64，hy2版本2.5.2
客户端：sing-box 1.11.0-alpha.10 AMD 64 windows 11
刚开始还以为是配置问题，仔细检查和比对了配置好久，还测试了服务器是否封禁udp；
最后在一个海外的机器上使用用同样的客户端配置发现可以正常联通，然后在本地换了个网络再次尝试，果然，电信网络就是正常的，联通网络就无法连接。不知道在重庆的小伙伴有没有跟我遇到同样的问题。

**Comment by ghost** [comment]
如果您使用的是sing-box作为客户端，那么请您试试关闭SNI后还会不会出现这种情形。只需要修改hysteria2出站的tls字段就可以。

https://sing-box.sagernet.org/zh/configuration/outbound/hysteria2
https://sing-box.sagernet.org/zh/configuration/shared/tls/#_2

**Reply by PekingJam** [reply]
刚才试了一下，添加了disable_sni: true，联通网络还是无法正常使用，然后电信也无法使用了。但是把disable_sni设为false，电信可以使用

**Comment by wolfid1** [comment]
没任何问题，其它协议在河南用不了，这玩意在河南都能用，重庆基本算不管的。你自己的问题吧
